[2023-11-25T23:11:29.322+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_Pipeline.load_to_gcp scheduled__2023-11-25T22:00:00+00:00 [queued]>
[2023-11-25T23:11:29.329+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_Pipeline.load_to_gcp scheduled__2023-11-25T22:00:00+00:00 [queued]>
[2023-11-25T23:11:29.329+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-25T23:11:29.341+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_to_gcp> on 2023-11-25 22:00:00+00:00
[2023-11-25T23:11:29.349+0000] {standard_task_runner.py:57} INFO - Started process 2264 to run task
[2023-11-25T23:11:29.352+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ETL_Pipeline', 'load_to_gcp', 'scheduled__2023-11-25T22:00:00+00:00', '--job-id', '438', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Pipeline.py', '--cfg-path', '/tmp/tmps37ziqk7']
[2023-11-25T23:11:29.355+0000] {standard_task_runner.py:85} INFO - Job 438: Subtask load_to_gcp
[2023-11-25T23:11:29.392+0000] {task_command.py:416} INFO - Running <TaskInstance: ETL_Pipeline.load_to_gcp scheduled__2023-11-25T22:00:00+00:00 [running]> on host 4012deabc79b
[2023-11-25T23:11:29.449+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETL_Pipeline' AIRFLOW_CTX_TASK_ID='load_to_gcp' AIRFLOW_CTX_EXECUTION_DATE='2023-11-25T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-25T22:00:00+00:00'
[2023-11-25T23:11:29.490+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/utils/helper.py", line 74, in load_to_gcp
    blob.upload_from_filename(local_file_path)
  File "/home/airflow/.local/lib/python3.8/site-packages/google/cloud/storage/blob.py", line 2926, in upload_from_filename
    self._handle_filename_and_upload(
  File "/home/airflow/.local/lib/python3.8/site-packages/google/cloud/storage/blob.py", line 2794, in _handle_filename_and_upload
    with open(filename, "rb") as file_obj:
FileNotFoundError: [Errno 2] No such file or directory: 'dags/data/scraped_data_2023-11-25.csv'
[2023-11-25T23:11:29.497+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=ETL_Pipeline, task_id=load_to_gcp, execution_date=20231125T220000, start_date=20231125T231129, end_date=20231125T231129
[2023-11-25T23:11:29.508+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 438 for task load_to_gcp ([Errno 2] No such file or directory: 'dags/data/scraped_data_2023-11-25.csv'; 2264)
[2023-11-25T23:11:29.523+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-25T23:11:29.537+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-25T23:14:29.074+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_Pipeline.load_to_gcp scheduled__2023-11-25T22:00:00+00:00 [queued]>
[2023-11-25T23:14:29.088+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_Pipeline.load_to_gcp scheduled__2023-11-25T22:00:00+00:00 [queued]>
[2023-11-25T23:14:29.088+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-25T23:14:29.103+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_to_gcp> on 2023-11-25 22:00:00+00:00
[2023-11-25T23:14:29.110+0000] {standard_task_runner.py:57} INFO - Started process 2329 to run task
[2023-11-25T23:14:29.113+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ETL_Pipeline', 'load_to_gcp', 'scheduled__2023-11-25T22:00:00+00:00', '--job-id', '443', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Pipeline.py', '--cfg-path', '/tmp/tmpn0wseg9r']
[2023-11-25T23:14:29.116+0000] {standard_task_runner.py:85} INFO - Job 443: Subtask load_to_gcp
[2023-11-25T23:14:29.159+0000] {task_command.py:416} INFO - Running <TaskInstance: ETL_Pipeline.load_to_gcp scheduled__2023-11-25T22:00:00+00:00 [running]> on host 4012deabc79b
[2023-11-25T23:14:29.212+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETL_Pipeline' AIRFLOW_CTX_TASK_ID='load_to_gcp' AIRFLOW_CTX_EXECUTION_DATE='2023-11-25T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-25T22:00:00+00:00'
[2023-11-25T23:14:29.492+0000] {helper.py:76} INFO - File news_data.csv uploaded to sementic_assignment successfully!
[2023-11-25T23:14:29.492+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-25T23:14:29.525+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ETL_Pipeline, task_id=load_to_gcp, execution_date=20231125T220000, start_date=20231125T231429, end_date=20231125T231429
[2023-11-25T23:14:29.569+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-25T23:14:29.588+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-25T23:16:08.366+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_Pipeline.load_to_gcp scheduled__2023-11-25T22:00:00+00:00 [queued]>
[2023-11-25T23:16:08.384+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_Pipeline.load_to_gcp scheduled__2023-11-25T22:00:00+00:00 [queued]>
[2023-11-25T23:16:08.385+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-25T23:16:08.407+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_to_gcp> on 2023-11-25 22:00:00+00:00
[2023-11-25T23:16:08.420+0000] {standard_task_runner.py:57} INFO - Started process 2369 to run task
[2023-11-25T23:16:08.423+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ETL_Pipeline', 'load_to_gcp', 'scheduled__2023-11-25T22:00:00+00:00', '--job-id', '448', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Pipeline.py', '--cfg-path', '/tmp/tmpbr4n8q7t']
[2023-11-25T23:16:08.426+0000] {standard_task_runner.py:85} INFO - Job 448: Subtask load_to_gcp
[2023-11-25T23:16:08.498+0000] {task_command.py:416} INFO - Running <TaskInstance: ETL_Pipeline.load_to_gcp scheduled__2023-11-25T22:00:00+00:00 [running]> on host 4012deabc79b
[2023-11-25T23:16:08.599+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETL_Pipeline' AIRFLOW_CTX_TASK_ID='load_to_gcp' AIRFLOW_CTX_EXECUTION_DATE='2023-11-25T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-25T22:00:00+00:00'
[2023-11-25T23:16:08.947+0000] {helper.py:76} INFO - File news_data.csv uploaded to sementic_assignment successfully!
[2023-11-25T23:16:08.947+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-25T23:16:08.968+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ETL_Pipeline, task_id=load_to_gcp, execution_date=20231125T220000, start_date=20231125T231608, end_date=20231125T231608
[2023-11-25T23:16:09.014+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-25T23:16:09.028+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-25T23:17:43.110+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_Pipeline.load_to_gcp scheduled__2023-11-25T22:00:00+00:00 [queued]>
[2023-11-25T23:17:43.123+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_Pipeline.load_to_gcp scheduled__2023-11-25T22:00:00+00:00 [queued]>
[2023-11-25T23:17:43.123+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-25T23:17:43.145+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_to_gcp> on 2023-11-25 22:00:00+00:00
[2023-11-25T23:17:43.154+0000] {standard_task_runner.py:57} INFO - Started process 2409 to run task
[2023-11-25T23:17:43.159+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ETL_Pipeline', 'load_to_gcp', 'scheduled__2023-11-25T22:00:00+00:00', '--job-id', '453', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Pipeline.py', '--cfg-path', '/tmp/tmp9162zq61']
[2023-11-25T23:17:43.161+0000] {standard_task_runner.py:85} INFO - Job 453: Subtask load_to_gcp
[2023-11-25T23:17:43.215+0000] {task_command.py:416} INFO - Running <TaskInstance: ETL_Pipeline.load_to_gcp scheduled__2023-11-25T22:00:00+00:00 [running]> on host 4012deabc79b
[2023-11-25T23:17:43.325+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETL_Pipeline' AIRFLOW_CTX_TASK_ID='load_to_gcp' AIRFLOW_CTX_EXECUTION_DATE='2023-11-25T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-25T22:00:00+00:00'
[2023-11-25T23:17:43.639+0000] {helper.py:76} INFO - File news_data_2023-11-25.csv uploaded to sementic_assignment successfully!
[2023-11-25T23:17:43.639+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-25T23:17:43.658+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ETL_Pipeline, task_id=load_to_gcp, execution_date=20231125T220000, start_date=20231125T231743, end_date=20231125T231743
[2023-11-25T23:17:43.734+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-25T23:17:43.757+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-25T23:31:03.164+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_Pipeline.load_to_gcp scheduled__2023-11-25T22:00:00+00:00 [queued]>
[2023-11-25T23:31:03.170+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_Pipeline.load_to_gcp scheduled__2023-11-25T22:00:00+00:00 [queued]>
[2023-11-25T23:31:03.170+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-25T23:31:03.181+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_to_gcp> on 2023-11-25 22:00:00+00:00
[2023-11-25T23:31:03.187+0000] {standard_task_runner.py:57} INFO - Started process 2627 to run task
[2023-11-25T23:31:03.189+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ETL_Pipeline', 'load_to_gcp', 'scheduled__2023-11-25T22:00:00+00:00', '--job-id', '458', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Pipeline.py', '--cfg-path', '/tmp/tmpd1fykjh4']
[2023-11-25T23:31:03.191+0000] {standard_task_runner.py:85} INFO - Job 458: Subtask load_to_gcp
[2023-11-25T23:31:03.225+0000] {task_command.py:416} INFO - Running <TaskInstance: ETL_Pipeline.load_to_gcp scheduled__2023-11-25T22:00:00+00:00 [running]> on host 4012deabc79b
[2023-11-25T23:31:03.286+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETL_Pipeline' AIRFLOW_CTX_TASK_ID='load_to_gcp' AIRFLOW_CTX_EXECUTION_DATE='2023-11-25T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-25T22:00:00+00:00'
[2023-11-25T23:31:03.644+0000] {helper.py:76} INFO - File news_data_2023-11-25.csv uploaded to sementic_assignment successfully!
[2023-11-25T23:31:03.644+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-25T23:31:03.662+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ETL_Pipeline, task_id=load_to_gcp, execution_date=20231125T220000, start_date=20231125T233103, end_date=20231125T233103
[2023-11-25T23:31:03.722+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-25T23:31:03.739+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
